src/index.ts

```diff
@@ -40,8 +43,10 @@ if (LOCAL) {
   }
 }
 
-// Constants
-const RANDOM_ACTIVITY_CRON = '0 0 0 * * *';
+// Define global state
+// TODO define a global state manager that all resources can pull from
```

Comment:
Something to keep in mind, this global state would be representing the bot state across all of the guilds it is in. Depending on the use case, it might be necessary to have that state be a map based on that (though I don't know how DMs would be handled in that case)